mustang_codegen/lib/src/service_generator/screen_service_generator.dart

```diff
@@ -121,7 +121,7 @@ class ScreenServiceGenerator extends Generator {
     List<String> appEventModels = [CodeGenConstants.defaultMustangModel];
     List<String> appEventModelImports = [];
     await for (AssetId assetId
-        in buildStep.findAssets(Glob('$appModelsDir/*[!.model.]*'))) {
+        in buildStep.findAssets(Glob('$appModelsDir/*'))) {
```

Comment:
won't this iterate through all the files? when all it has to look at are `*.model.dart` files?

Reply:
appModelsDir is `lib/src/models`. Now that generated files are out of this, wildcard does the job